cli/credentials.go

```diff
@@ -1,88 +1,29 @@
-package keyconjurer
+package main
 
 import (
-	"encoding/json"
 	"fmt"
 	"os"
-	"strconv"
+	"runtime"
+	"strings"
 	"time"
+
+	ps "github.com/mitchellh/go-ps"
 )
 
-// Credentials are used to store and print out temporary AWS Credentials
+// AWSCredentials are used to store and print out temporary AWS Credentials
 // Note: verified that onelogin uses int as ID (so no leading 0's)
 // ... but does mean we can have negative user ids
-type Credentials struct {
-	accountId       uint
+type AWSCredentials struct {
+	AccountID       string `json:"AccountId"`
 	AccessKeyID     string `json:"AccessKeyId"`
 	SecretAccessKey string `json:"SecretAccessKey"`
 	SessionToken    string `json:"SessionToken"`
 	Expiration      string `json:"Expiration"`
 }
 
-func GetCredentials(u *UserData, accountName string, ttl uint) (*Credentials, error) {
-	// check if account is an account currently assigned
-	accountFound, err := u.FindAccount(accountName)
-	if err != nil {
-		return nil, err
-	}
-
-	// check if account asked for is in ENV
-	// and still valid
-	//
-	// on false always build new Credential
-	var credentials *Credentials
-	switch envCredsValid(accountFound, u.TimeRemaining) {
```

Comment:
This is broken and needs to be fixed. the `--time-remaining` flag is broken which means users will always be prompted for new credentials every time they call get, even if their credentials are still valid.